src/web/components/ParticipantRequests/ParticipantRequestsTable.tsx

```diff
@@ -30,6 +30,7 @@ export function ParticipantRequestsTable({
 }: ParticipantRequestsTableProps) {
   return (
     <div className='participant-requests-container'>
+      <h2>{participantRequests.length} Participants Pending Approval</h2>
```

Comment:
Let's conditionally render the string to be singular when there is only one `participantRequest` to avoid 
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/61671db1-81ca-428b-897e-19782fcedd26)